cluster/manifests/flannel/daemonset.yaml

```diff
@@ -19,9 +19,10 @@ spec:
         daemonset: kube-flannel
         application: kubernetes
         component: flannel
-        version: v0.18.1
+        version: v0.19.2
       annotations:
         logging/destination: "{{.Cluster.ConfigItems.log_destination_infra}}"
+        config/hash: {{"configmap.yaml" | manifestHash}}
```

Comment:
This makes the daemonset rotate when configmap is changed e.g by changing `cluster_cidr`